outputs.tf

```diff
@@ -32,11 +32,6 @@ output "subnet_group_arn" {
 }
 
 output "endpoint" {
-  description = "The connection endpoint in address:port format."
```

Comment:
I don't think this output is useful at all, and it will just confuse us when swapping between RDS instance and cluster (since `endpoint` in the cluster resource == `address` in the instance).

Reply:
@mikael-lindstrom and @lukaspour  - Thoughts on the comment/commit above?

Reply:
I think we could remove it, but I'm not a primary user of this module so I'm interested in @lukaspour comments.

Reply:
Yes I agree it can be confusing. The terminology should be consistent.

Reply:
will go ahead and merge this then 👍